testsuite/integration-arquillian/tests/pom.xml

```diff
@@ -224,7 +224,7 @@
         <firefoxUserPreferences/>
         <firefoxHeadless>true</firefoxHeadless>
         <chromeBinary/>
-        <chromeArguments>--headless</chromeArguments>
+        <chromeArguments>--headless --window-size=1920,1080</chromeArguments>
```

Comment:
Curious about the "window-size" why are you adding this?

Reply:
By default, it uses 800x600 and since it's in headless mode, fullscreen does not have any effect as there's no screen to use as full 😀
800x600 triggers mobile layout